code-challenges/javaChallenges/lib/src/test/java/javaChallenges/BinarySearchTest.java

```diff
@@ -0,0 +1,25 @@
+package javaChallenges;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class BinarySearchTest {
```

Comment:
Great work on the tests, just don't forget to check for the base case. What if you were given an empty array?